ci/cpu/build.sh

```diff
@@ -123,7 +123,7 @@ if [ "$BUILD_CUGRAPH" == "1" ]; then
     gpuci_logger "cugraph-service"
     gpuci_conda_retry mambabuild --no-build-id --croot ${CONDA_BLD_DIR} conda/recipes/cugraph-service --python=$PYTHON
     gpuci_logger "cugraph-pyg"
-    gpuci_conda_retry mambabuild --no-build-id --croot ${CONDA_BLD_DIR} conda/recipes/cugraph-pyg --python=$PYTHON
+    gpuci_conda_retry mambabuild --no-build-id --croot ${CONDA_BLD_DIR} conda/recipes/cugraph-pyg --python=$PYTHON -c pyg -c pytorch
```

Comment:
@alexbarghi-nv, for reference, this file was used by Jenkins, but is no longer in use by GH Actions.

editing it will have no effect on the current CI runs.

we will be opening PRs to remove these dead files across all the repositories after the `23.02` release.